EPPs/etc/example_clarity_script.yml

```diff
@@ -57,3 +57,20 @@ rest_api:
   url: localhost
   username: user
   password: password
+
```

Comment:
I would probably add a level to make it clear what this section is used for. 
Something like `preseqlab_assignments` or similar.
This will be useful when you need to add section for other scripts.

Reply:
Added additional level to categorise the step i.e. "start" of the workflow. This makes the config more relevant if different workflows will assign to the same step.